server/src/main/java/org/elasticsearch/Version.java

```diff
@@ -148,6 +148,8 @@
     public static final Version V_6_1_3 = new Version(V_6_1_3_ID, org.apache.lucene.util.Version.LUCENE_7_1_0);
     public static final int V_6_1_4_ID = 6010499;
     public static final Version V_6_1_4 = new Version(V_6_1_4_ID, org.apache.lucene.util.Version.LUCENE_7_1_0);
+    public static final int V_6_1_5_ID = 6010599;
+    public static final Version V_6_1_5 = new Version(V_6_1_5_ID, org.apache.lucene.util.Version.LUCENE_7_1_0);
```

Comment:
Let us leave adding 6.1.5 out of this PR, that should be picked up separately. The strategy would be:
 - add 6.1.5 to 6.1 (plus a version bump to 6.1.5), 6.2, 6.x, master
 - add 6.2.4 to 6.2 (plus a version bump to 6.2.4), 6.x, master